rotkehlchen/exchanges/bitfinex.py

```diff
@@ -324,16 +328,31 @@ def _api_query_paginated(
             results.extend(cast('Iterable', results_))
             # NB: Copying the set before updating it prevents losing the call args values
             processed_result_ids = processed_result_ids.copy()
-            # type ignore is due to always having a trade link for bitfinex trades
-            processed_result_ids.update({int(result.link) for result in results_})  # type: ignore
+            newly_processed_results = set()
+            for result in results_:
+                if isinstance(result, AssetMovement):
+                    if result.unique_id is None:
+                        continue  # fees don't have a unique id
+
+                    newly_processed_results.add(int(result.unique_id))
+                else:
+                    # type ignore is due to always having a trade link for bitfinex trades
+                    newly_processed_results.add(int(result.link))  # type: ignore
```

Comment:
same here. What does casting to int mean?

Reply:
As mentioned in the other comment, this was cast to int so it would match the raw data. Handling this as a string now.